src/containers/shared/components/Transaction/OracleSet/test/ConvertScalePrice.test.ts

```diff
@@ -0,0 +1,26 @@
+import { convertScaledPrice } from '../parser'
+
+describe('convertScaledPrice', () => {
+  it('should convert scaled price to original price correctly using scale factor', () => {
+    // scaled price: 5083
```

Comment:
If you want this to be easier to read, you can convert the numbers to hex in the test itself instead of just using the hex